### PR TITLE
chore(website): add September newsletter

### PR DESCRIPTION
**Pull request by gloriliale**
## In this PR
- Add September newsletter to the website blog
- Add the images used for the blog to our repo
- Add the new page to our website VRT

### CODE REVIEW

**Review by nkrantz — APPROVED** [review]

File: packages/paste-website/src/pages/blog/2022-09-23-paste-newsletter.mdx [hunk 1]
```diff
@@ -0,0 +1,456 @@
...
+      <Td>
+        Improvements to the existing <Anchor href="/components/combobox">combobox</Anchor> component, with a design
+        refresh and added functionality to select multiple items.
+      </Td>
+    </Tr>
+    <Tr>
+      <Td>
+        <strong>Avatar component, Square variant</strong>
+      </Td>
+      <Td>A new variant for the Avatar Component that creates a square shape.</Td>
```
Comment (loreina) [inline packages/paste-website/src/pages/blog/2022-09-23-paste-newsletter.mdx:355]: question: do you mind if we update this copy to saying `entity` variant? I took a v quick stab at the copy but feel free to change it up however

```suggestion
        <strong>Avatar component, entity variant</strong>
      </Td>
      <Td>A new variant for the Avatar Component for entities, like a business.</Td>
```

**Review by loreina — COMMENTED** [review]
looks great! just left a quick comment

**Review by nkrantz — APPROVED** [review]

**Review by SiTaggart — APPROVED** [review]

File: packages/paste-website/src/pages/blog/2022-09-23-paste-newsletter.mdx [hunk 2]
```diff
@@ -0,0 +1,456 @@
...
+</Paragraph>
+
+#### 42 new icons, including a full suite of contact center icons!
+
+![](../../assets/images/articles/2022-09-23-paste-newsletter/new-icons-example.png)
+
+We&apos;ve got new icons! A full suite of contact center icons and more are now available.
+Thanks to <strong>Fazle Jahangir, Isabela Razetti,</strong> and <strong>Porlin Chin</strong> for their close collaboration on these icons.
+
+<br />
```
Comment (serifluous) [inline packages/paste-website/src/pages/blog/2022-09-23-paste-newsletter.mdx:129]: todo: remove this br

Comment (gloriliale) [reply packages/paste-website/src/pages/blog/2022-09-23-paste-newsletter.mdx:129]: @serifluous I see you, but I'd like to push back on doing this in this PR. It's a layout thing as opposed to something wrong with the content, and we've delayed the newsletter rollout already from the email restrictions. I'd rather fix this is another PR on the queue.

File: packages/paste-website/src/pages/blog/2022-09-23-paste-newsletter.mdx [hunk 3]
```diff
@@ -0,0 +1,456 @@
...
+*View the full list at the end of this newsletter.
+
+<Paragraph>
+  <Anchor href="/components/icons">Search through our icons list</Anchor>
+</Paragraph>
```
Comment (serifluous) [inline packages/paste-website/src/pages/blog/2022-09-23-paste-newsletter.mdx:134]: suggestion: Swap this to

`[View the full list of new icons](link to the list at the bottom of page) and [search through our full icons list](/components/icons)`

File: packages/paste-website/src/pages/blog/2022-09-23-paste-newsletter.mdx [hunk 4]
```diff
@@ -0,0 +1,456 @@
...
+      </Td>
+    </Tr>
+  </TBody>
+</Table>
+
+<Box marginBottom="space120" />
+
+## 💬 Customer insights
+
+The Design Systems team keeps a pulse on Paste users with our [annual CSAT survey](https://airtable.com/appiDf1K8tPC1fPoJ/pagAqvPc6EWGFAzYO) annual CSAT survey.
```
Comment (serifluous) [inline packages/paste-website/src/pages/blog/2022-09-23-paste-newsletter.mdx:215]: todo: remove "annual CSAT survey"

**Review by TheSisb — DISMISSED** [review]

**Review by serifluous — APPROVED** [review]
There are some small things that need to be changed. Will list them somewhere else to fix in a later PR.

**Review by nkrantz — APPROVED** [review]

**Review by TheSisb — APPROVED** [review]

**Review by serifluous — APPROVED** [review]

**Pull request merged**